### PR TITLE
feat: add usip document

### PR DESCRIPTION
**Pull request by zsq1234**

### CODE REVIEW

File: src/docs/guides/general/server/usip.en-US.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+USIP refers to the protocol that must be followed when integrating a third-party system with Univer Server, specifically requiring implementation of a series of USIP-defined interfaces.
+
+Based on these protocols, Univer Server can access third-party systems to obtain the results required by clients.
+
+In this way, while ensuring that data remains within the third-party client system, Univer Server can successfully integrate into the client's system.
+
+![relationship](./usip/AzQSbQDVuoKkRZxrRPqcHtF0nsh.png)
+![api call flowchart](./usip/NdzmbFpnio4mQexalKBcw7Hwnmf.png)
+
+# Prerequisites
```
Comment (jikkai) [inline src/docs/guides/general/server/usip.en-US.mdx:19]: The title should have the correct level.

File: src/docs/guides/general/server/usip.en-US.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+```shell
+curl -X GET \
+    -H "authorization: xx" \
+    -H "cookie: xx" \
+    -H "YOUR-CUSTOM-HEADER: xx" \
+    "http://localhost:8080/credential"
+    
+# response
+{
```
Comment (jikkai) [inline src/docs/guides/general/server/usip.en-US.mdx:39]: In documentation, we use two spaces in the code block.

**Review by jikkai — CHANGES_REQUESTED** [review]

**Review by hexf00 — APPROVED** [review]

File: src/docs/guides/general/server/usip.zh-CN.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+# response
+{
+  "users": [
+    {"userID": "1", "name":"xxx", "avatar":"https://xxxx"},
+    {"userID": "2", "name":"xxx", "avatar":"https://xxxx"},
+  ]
+}
+```
+
+3. Permission role： 获取特定用户对特定文档的权限 (AuthZ)
```
Comment (jikkai) [inline src/docs/guides/general/server/usip.zh-CN.mdx:67]: There should not be an English space after Chinese punctuation marks.

File: src/docs/guides/general/server/usip.zh-CN.mdx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+![relationship](./usip/usip-relationship.png)
+![api call flowchart](./usip/usip-flowchart.png)
+
+## 前提
+
+1. 集成方需要自己维护 用户 - 文档 之间的关系，univer server 作为客户端通过接口获取文档下的用户权限
+2. 实现：调用 univer server 创建文档成功后，会返回一个 `unitID` 文档 id，集成方需要对其进行存储，以实现自己的文件管理业务
+
+## **需要实现的接口**
```
Comment (jikkai) [inline src/docs/guides/general/server/usip.zh-CN.mdx:26]: Is it really necessary to use bold in the title?

**Review by jikkai — CHANGES_REQUESTED** [review]

File: src/docs/guides/general/server/usip.zh-CN.mdx [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+1. Credential verify: 验证用户身份合法性 (AuthN)
```
Comment (jikkai) [inline src/docs/guides/general/server/usip.zh-CN.mdx:28]: There is a mix of Chinese and Western punctuation marks here.

**Review by jikkai — CHANGES_REQUESTED** [review]

**Review by jikkai — APPROVED** [review]

**Pull request merged**